### DISCUSSION TITLE
How to customize the Telescope layout?

### DISCUSSION THREAD

**Discussion by chrisgrieser** [pwntester/octo.nvim#416] [Q&A] [answered]
Thanks for this awesome plugin!

So I was wondering whether it is possible to customize the Telescope layout? As far as I can tell, octo's Telescope commands simply inherit the default layout from Telescope. 

However, with things like GitHub issues, I'd prefer using a different layout. (e.g. for files, I like horizontal, but I prefer vertical for issues).

The docs do not mention anything about customizing layouts. I assume octo is setting up Telescope extensions in the background, so I tried my luck configuring an `octo` extension in Telescope's settings, but that also didn't work.

**Comment by pwntester** [comment, marked as answer]
Hi!

You can do that by passing any layout config to the `issues` command. eg: `lua require('octo.pickers.telescope.provider').issues({layout_strategy='vertical',layout_config={width=0.5}})`

There is no way to configure this so you need to create an alias for that command.

You could also propose a PR where layouts or other telescope options could be configured and passed to each command. Those config options could be merged into the `opts` before calling the picker commands. eg: [here](https://github.com/pwntester/octo.nvim/blob/master/lua/octo/commands.lua#L42)
